activerecord/test/cases/validations/uniqueness_validation_test.rb

```diff
@@ -35,6 +36,12 @@ class TopicWithUniqEvent < Topic
   validates :event, uniqueness: true
 end
 
+class MateyWithUniqValidation < Matey
```

Comment:
All the specs here deal with uniq validation. This class specificially should be something like `MateyWithoutPrimaryKey`